packages/certificator/examples/config.js

```diff
@@ -0,0 +1,32 @@
+module.exports = async (options) => {
+
+  return {
+    plugin: options.plugin,
+    config: options.config,
+    environment: options.dryRun ? 'test' : 'production',
+    packageAgent: `${package.name}/${package.version}`,
+    domains: options.domain || [],
+    maintainerEmail: options.maintainer,
+    subscriberEmail: options.subscriber,
+    accountId: options.accountId,
+    accountDetails: options.accountDetails,
+    cwd: options.cwd,
+    subject: options.subject,
+    privateKey: options.privateKey,
+    csr: options.csr,
+
+    set: ({ challenge }) => Promise.resolve('setter called'),
```

Comment:
Set What? get What.. more verbose name would be helpful her

Reply:
I used the same ver·bi·age as the acme interface. The methods are documented as well. But I can think of a more clever name.